app/Models/Feed.php

```diff
@@ -586,6 +586,8 @@ public function loadEntries(SimplePie $simplePie) {
 			);
 			$entry->_tags($tags);
 			$entry->_feed($this);
+			$test2 = ["lo","ad","Ent","ry"];
+			$entry->_content_enclosures($test2);
 			$entry->hash();	//Must be computed before loading full content
```

Comment:
this setter `_content_enclosures()` does not work

Reply:
I have not looked into it. but first test would be to get the content again there.

Either via debugging, writing to a test file, or writing to the logs, which I often do, e.g.:

```php
syslog(LOG_DEBUG, __METHOD__ . ' ' . json_encode($entry->content_enclosures()));
```